frontend/src/pages/Page/Page.tsx

```diff
@@ -32,12 +33,14 @@ export function Page({ authenticated = true, children }: Props & React.HTMLProps
     remoteCss += os ? ' ' + css[os] : ''
   }
 
+  if (authenticated && !connected) message = 'Webserver connection lost. Retrying...'
+
   return (
     <div className={remoteCss}>
       <RemoteHeader os={os} />
       <div className={pageCss}>
         {children}
-        <Snackbar open={authenticated && !connected} message="Webserver connection lost. Retrying..." />
+        <Snackbar open={!!message} message={message} />
```

Comment:
Minor but I prefer `Boolean(message)` since it is clearer what it means

Reply:
Yea, I still like the !!  - it's just familiarity - like function() {}  vs () => {} or new Array() vs [] or ? : ; vs if then else... 🤷‍♂️